docset/winserver2022-ps/failoverclusters/Add-ClusterVirtualMachineRole.md

```diff
@@ -75,7 +77,7 @@ Name                                    OwnerNode
 print-VM1                               node1                                   Online
 ```
 
-This example queries for virtual machines matching the wildcard characters *print* and configures
+This example queries for virtual machines matching the wildcard characters **print** and configures
```

Comment:
I think contextually this is meant to show the wildcards rather than emphasize the term.

```suggestion
This example queries for virtual machines matching the wildcard characters `*print*` and configures
```